wimm.Secundatives/Range.cs

```diff
@@ -26,8 +26,8 @@ public class Range<T> : IRange<T> where T : IComparable<T>
         /// </exception>
         public Range(T min, T max)
         {
-            Min = Require(min, nameof(min));
-            Max = Require(max, nameof(max));
+            Min = Range<T>.Require(min, nameof(min));
```

Comment:
Is qualification required here?

Reply:
I'll check. This might have just been compiler fighting